src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs

```diff
@@ -49,5 +59,21 @@ public override string ToString()
         {
             return $"Key: {Key}, Path: {Path}, VR:{VR}, Level:{Level}, Status:{Status}";
         }
+
+        public override int GetHashCode()
+        {
+            return HashCode.Combine(Key.GetHashCode(), Path, VR, Level.GetHashCode(), Status.GetHashCode());
+        }
+
+        public override bool Equals(object obj)
```

Comment:
it's good to have GetHashCode and Equals, just curious  where we use them?

Reply:
The Equals requires GetHashCode and is used in test code to compare custom tag entries.

---
In reply to: [558595543](https://github.com/microsoft/dicom-server/pull/501#discussion_r558595543) [](ancestors = 558595543)